src/github.com/getlantern/lantern-mobile/app/build.gradle

```diff
@@ -1,20 +1,45 @@
 apply plugin: 'com.android.application'
 apply plugin: 'io.fabric'
 
+def incVersionNum() {
+ 	def versionPropsFile = file('version.properties')
+	def Properties versionProps = new Properties()
+
+    // only try to load properties if version.properties 
+    // file exists
+   if (versionPropsFile.exists()) {
+		versionProps.load(new FileInputStream(versionPropsFile))
+	}
+    
+    def value = 0
+    def runTasks = gradle.startParameter.taskNames
+    if ('assembleRelease' in runTasks) {
+        value = 1
+    }
+
+    def code = (versionProps['VERSION_CODE'] ?: "0").toInteger() + value
+    versionProps['VERSION_CODE'] = code.toString()
+    versionProps.store(versionPropsFile.newWriter(), null)
+
+    return code
+}
```

Comment:
The version code is automatically incremented for every new release build. This gets written to the ``version.properties`` file (which is created and defaults to zero if it doesn't exist already).